extensions/wikia/UserRenameTool/RenameUserProcess.class.php

```diff
@@ -1068,27 +698,102 @@ static public function newFromData( $data ) {
 	}
 
 
-	/* Static utility functions */
+	/**
+	 * Marshal & execute the RenameUserProcess functions to rename a user
+	 *
+	 * @param array $params
+	 *		requestor_id => ID of the user requesting this rename action
+	 *		requestor_name => Name of the user requesting this rename action
+	 *		rename_user_id => ID of the user to rename
+	 *		rename_old_name => Current username of the user to rename
+	 *		rename_new_name => New username for the user to rename
+	 *		reason => Reason for requesting username change
+	 *		rename_fake_user_id => Repeated rename process special case (TODO: Don't know what this is)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app